gen3-integration-tests/tests/test_indexing_page.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import requests
+import os
+import utils.gen3_admin_tasks as gat
+
+from utils import logger
+from gen3.auth import Gen3Auth
+from gen3.index import Gen3Index
+from pages.login import LoginPage
+from pages.indexing_page import IndexingPage
+from services.indexd import Indexd
+
+
+@pytest.mark.portal
+@pytest.mark.sower
+@pytest.mark.ssjdispatcher
+class TestIndexingPage:
+    variables = {}
+
+    def setup_class(cls):
+        cls.variables["test_guid"] = "c2da639f-aa25-4c4d-8e89-02a143788268"
+        cls.variables["test_hash"] = (
+            "73d643ec3f4beb9020eef0beed440ad4"  # pragma: allowlist secret
+        )
+        cls.variables["expected_result"] = (
+            f"{cls.variables['test_guid']},"
+            "s3://cdis-presigned-url-test/testdata,,jenkins2,"
+            f"{cls.variables['test_hash']},13,"
+        )  # pragma: allowlist secret
```

Comment:
instead of `pragma: allowlist secret`, add the known false positives to the secrets baseline:

`detect-secrets scan --baseline .secrets.baseline`

Reply:
👍